README.md

```diff
@@ -93,6 +93,21 @@ $ grunt prod
 
 * explore `config/env/production.js` for production environment configuration options
 
+### Running with User Seed
+To have an 'admin' and 'user' user account seeded at runtime:
+
+In Development:
+```bash
+MONGO_SEED=true grunt
+```
+This will seed the users everytime, just grab the password from the console.
```

Comment:
"It will try to seed the users everytime. If one of the user already exists, it will display an error message on the console. Just grab the passwords from the console"

https://github.com/meanjs/mean/blob/ea97f2e7d83281a5bf34cb294ce9f1483135dbf2/config/lib/seed.js#L55